packages/core-ethereum/src/index.ts

```diff
@@ -124,6 +134,33 @@ export default class HoprEthereum {
     return await this.indexer.getPublicNodes()
   }
 
+  private async setInitialCommitmentIfNotSet(channel: ChannelEntry): Promise<void> {
```

Comment:
at some point we need to instantiate channel and then set the commitment. Currently HoprEthereum is instantiated and connected to the indexer from which it will consume the ChannelUpdate events, afterwards we can instantiate the subsequent classes, namely Channel and Commitment

Reply:
we have channel.commitment.initialize() - why not use that directly?